pandas/tests/frame/test_indexing.py

```diff
@@ -204,6 +204,15 @@ def test_setitem_list_of_tuples(self, float_frame):
         expected = Series(tuples, index=float_frame.index, name="tuples")
         tm.assert_series_equal(result, expected)
 
+    def test_setitem_single_row_categorical(self):
+        df = DataFrame({"Alpha": ["a"], "Numeric": [0]})
+        categories = pd.Categorical(df["Alpha"], categories=["a", "b", "c"])
+        df.loc[:, "Alpha"] = categories
```

Comment:
can you move this to the class (same file): ```class TestDataFrameIndexingCategorical```

Reply:
Done